jobbergate-api/pyproject.toml

```diff
@@ -5,7 +5,7 @@ authors = [
 name = "jobbergateapi2"
 description = "jobbergate-api"
 license = "MIT"
-version = "0.1.0"
+version = "0.0.1"
```

Comment:
??????

Reply:
I did this so the first time we run the release script, we can bump the minor version. It's still unreleased, so this doesn't mean that much at this point. Poetry sets the initial version to 0.1.0 by default, but I want that to be the first version we actually release.

If it's too weird, I can just make 0.2.0 the first released version

Reply:
Let's not roll back versions, this can cause a lot of confusion.

Reply:
It's unreleased. It's not really a rollback.